src/main/java/com/patchworkmc/mapping/remapper/NaiveRemapper.java

```diff
@@ -34,13 +37,20 @@ public void acceptMethod(IMappingProvider.Member method, String dstName) {
 					return;
 				}
 
+				if (blacklistedMethods.contains(method.name)) {
+					Patchwork.LOGGER.debug("Another duplicated method mapping for %s (proposed %s; original not cached)", method.name, dstName);
```

Comment:
```suggestion
					Patchwork.LOGGER.debug("Another duplicated method mapping for %s (proposed %s)", method.name, dstName);
```

I think the "original not cached" part makes things more confusing